<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoadPaletteWidget</class>
 <widget class="QWidget" name="RoadPaletteWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Road 1 Palette</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Background</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="PreviewPalette" name="road1Back" native="true">
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Edge</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="PreviewPalette" name="road1Edge" native="true">
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Stripes</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="PreviewPalette" name="road1Stripe" native="true">
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Centre</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="PreviewPalette" name="road1Cent" native="true">
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="buttonCopyTo1">
          <property name="toolTip">
           <string>Copy Palette From Road 2 To Road 1</string>
          </property>
          <property name="text">
           <string>Copy from Road 2</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Road 2 Palette</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Background</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="PreviewPalette" name="road2Back" native="true">
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Edge</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="PreviewPalette" name="road2Edge" native="true">
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Stripes</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="PreviewPalette" name="road2Stripe" native="true">
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Centre</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="PreviewPalette" name="road2Cent" native="true">
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="buttonCopyTo2">
          <property name="toolTip">
           <string>Copy Palette From Road 1 To Road 2</string>
          </property>
          <property name="text">
           <string>Copy from Road 1</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QSpinBox" name="spinRoadPal">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Road Palette Index</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Sky Palette</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <item alignment="Qt::AlignHCenter|Qt::AlignTop">
         <widget class="PreviewPalette" name="sky" native="true">
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>128</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>128</width>
            <height>128</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QToolButton" name="buttonFill">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Right click on palette entry to select fill color</string>
            </property>
            <property name="text">
             <string>Fill</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttonInt">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Right click on palette entry to select start color</string>
            </property>
            <property name="text">
             <string>Interpolate</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
         <widget class="QSpinBox" name="spinSkyPal">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Sky Palette Index</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Ground Palette</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <item alignment="Qt::AlignHCenter|Qt::AlignTop">
         <widget class="PreviewPalette" name="ground" native="true">
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>128</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
         <widget class="QToolButton" name="buttonIntG">
          <property name="toolTip">
           <string>Interpolate from first to last color</string>
          </property>
          <property name="text">
           <string>Interpolate</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
         <widget class="QSpinBox" name="spinGndPal">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Ground Palette Index</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PreviewPalette</class>
   <extends>QWidget</extends>
   <header>previewpalette.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
